pkg/csi/plugins/nodeserver.go

```diff
@@ -137,7 +138,23 @@ func (ns *nodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublis
 		mountPath = fluidPath + "/" + subPath
 	}
 
-	// 1. Wait the runtime fuse ready and check the sub path existence
+	// 1. Wait for mountPath to be created by mountPod
+	if err := wait.PollUntilContextTimeout(ctx, 2*time.Second, 30*time.Second, true, func(ctx context.Context) (done bool, err error) {
+		if _, err := os.Stat(mountPath); err != nil {
```

Comment:
From the perspective of the blast radius and the startup time, is it necessary to check whether the path is created only for CheckMountReadyMountMode?